Cargo.toml

```diff
@@ -280,6 +280,12 @@ inherits = "dev"
 debug = 0
 strip = "debuginfo"
 
+# Profile for uv-build
+[profile.minimal-size]
+inherits = "release"
+opt-level = "z"
+panic = "abort"
+
```

Comment:
You could see if this helps:

- https://github.com/johnthagen/min-sized-rust?tab=readme-ov-file#reduce-parallel-code-generation-units-to-increase-optimization

```toml
codegen-units = 1
```